WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelPackageList: View {
+    @EnvironmentObject private var viewModel: ShippingLabelPackageDetailsViewModel
+    @Environment(\.presentationMode) var presentation
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 0) {
+
+                /// Custom Packages
+                ///
+                if viewModel.customPackages.count > 0 {
+                    ListHeaderView(text: Localization.customPackageHeader.uppercased(), alignment: .left)
+                        .background(Color(.listBackground))
+                }
+                ForEach(viewModel.customPackages, id: \.title) { package in
+                    let selected = package == viewModel.selectedCustomPackage
+                    SelectableItemRow(title: package.title, subtitle: package.dimensions + " \(viewModel.dimensionUnit)", selected: selected).onTapGesture {
```

Comment:
Not a big deal, but probably a good idea to encapsulate `package.dimensions + " \(viewModel.dimensionUnit)"` into the view model, probably as a function!